qpsolvers/solver_output.py

```diff
@@ -0,0 +1,34 @@
+from dataclasses import dataclass
```

Comment:
```suggestion
#!/usr/bin/env python
# -*- coding: utf-8 -*-
#
# Copyright (C) 2016-2022 The qpsolvers contributors.
#
# This file is part of qpsolvers.
#
# qpsolvers is free software: you can redistribute it and/or modify it under
# the terms of the GNU Lesser General Public License as published by the Free
# Software Foundation, either version 3 of the License, or (at your option) any
# later version.
#
# qpsolvers is distributed in the hope that it will be useful, but WITHOUT ANY
# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR
# A PARTICULAR PURPOSE.  See the GNU Lesser General Public License for more
# details.
#
# You should have received a copy of the GNU Lesser General Public License
# along with qpsolvers. If not, see <http://www.gnu.org/licenses/>.

"""
Type for extended QP solver outputs going beyond the primal solution.
"""

from dataclasses import dataclass
```
Same header as other files of the project. The module docstring gives some high-level overview... It also avoids a linter warning :wink: